contracts/interfaces/IInvariantValidator.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 pragma solidity 0.8.2;
 
-import { BassetData, InvariantConfig } from "../masset/MassetStructs.sol";
+import "../masset/MassetStructs.sol";
```

Comment:
Any reason not to use explicit imports? I prefer implicit imports as it makes it easier to trace where contracts, interfaces, structs etc come from. But it doesn't really matter so no need to change.

Reply:
`MassetStructs` for me is an exception given that there are so many individual structs there